example: Flash all LED on

rzr opened the pull request:
This is helpful for testing if all LED are working same

Change-Id: Ifa7680661045dd07b80df9b0c6cf34c9f62e5cb8
Origin: https://github.com/TizenTeam/node-blinkt
Signed-off-by: Philippe Coval <philippe.coval@osg.samsung.com>

rzr committed:
example: Fade all LEDs on

This is helpful for testing if all LED are working same

Change-Id: Ifa7680661045dd07b80df9b0c6cf34c9f62e5cb8
Origin: https://github.com/TizenTeam/node-blinkt
Forwarded: https://github.com/Irrelon/node-blinkt/pull/14
Signed-off-by: Philippe Coval <philippe.coval@osg.samsung.com>

Irrelon commented:
Hey ya, would you like to move your changes into a new example-flash.js and keep the existing example.js as it was? I love the new example but want to keep the basic example as absolutely clear and simple as possible. Many people coming new to JS will find the setInterval confusing as well as the manipulation of the LED colours... I'd like the initial example to be readable by a 5 - 7 year old who is interested in programming but may not understand things like % operator or ++.

rzr committed:
example: Relocate to example subdir

Also support 'npm test'.

More examples to come later.

Change-Id: Ib4541c51b42584da74a5b81ee32747989d377d20
Forwarded: https://github.com/Irrelon/node-blinkt/pull/14
Signed-off-by: Philippe Coval <p.coval@samsung.com>

rzr commented:
Review welcome

cc: @ziransun

Irrelon commented:
Wonderful, thanks so much for your time to make these improvements.
